tapir/shifts/templates/shifts/shift_calendar_template.html

```diff
@@ -10,56 +10,79 @@
     <script src="{% static 'shifts/ShiftFiltersManager.js' %}?version=2" defer></script>
 {% endblock head %}
 {% block content %}
-    <div class="card-group mb-2">
-        <div class="card">
-            <h5 class="card-header">
-                {% block card_title %}
-                {% endblock card_title %}
-            </h5>
-            <div class="card-body">
-                {% block card_helptext %}
-                {% endblock card_helptext %}
+    <div class="d-flex align-items-end flex-column">
+        <button class="btn btn-link"
+                data-bs-toggle="collapse"
+                data-bs-target="#collapseCalendarInfo"
+                aria-expanded="false"
+                style="text-decoration: none">
+            <div class="d-flex align-items-center" style="text-decoration: none">
```

Comment:
`div` is not allowed inside `button`, your IDE probably gives a warning about that.